libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -579,7 +1107,91 @@ async def create(
         webhook: Optional[str] = None,
         multitask_strategy: Optional[MultitaskStrategy] = None,
     ) -> Run:
-        """Create a background run."""
+        """Create a background run.
+
+        Args:
+            thread_id (optional,str): the thread ID to assign to the thread. Defaults to None.
+                If None a random UUID will be generated.
```

Comment:
this is actually not the point in this case -- this is for creating stateless runs. e.g. if you pass thread_id = None, we won't store the thread state. the difference from `wait` is that it won't return any output, but can still be useful for side effects, i.e. agents modifying some external database